Tests.Integration.Presentation.Web/UI_Configuration/UIConfigurationTests.cs

```diff
@@ -17,32 +17,28 @@ public class UIConfigurationTests : WithAutoFixture
         [Fact]
         public async Task Can_Put()
         {
-            //Arrange
             var module = A<string>();
-            var key = "Module.Key";
-            var isEnabled = false;
-
-            var body = new UIModuleCustomizationDTO
-            {
-                Nodes = new List<CustomizedUINodeDTO>
-                {
-                    new()
-                    {
-                        Key = key,
-                        Enabled = isEnabled
-                    }
-                }
-            };
 
+            await UIConfigurationHelper.CreateUIModuleAndSaveAsync(TestEnvironment.DefaultOrganizationId, module);
+        }
+
+        [Fact]
+        public async Task Can_Get()
+        {
+            //Arrange
+            var module = A<string>();
+            
             //Act
-            using var response = await UIConfigurationHelper.SendPutRequestAsync(TestEnvironment.DefaultOrganizationId, module, body);
+            await UIConfigurationHelper.CreateUIModuleAndSaveAsync(TestEnvironment.DefaultOrganizationId, module);
+            
+            var data = await UIConfigurationHelper.GetCustomizationByModuleAsync(TestEnvironment.DefaultOrganizationId, module);
 
             //Assert
-            Assert.Equal(HttpStatusCode.NoContent, response.StatusCode);
+            Assert.NotNull(data);
```

Comment:
that's a bit poor... please make sure that the keys that you expect to be there are actually returned in the state that you expect